gradient_sim.py

```diff
@@ -137,346 +149,307 @@
-plt.title('%ECM = 100 %AM = 0 %clay = 10',fontsize=SMALL_SIZE)
-plt.legend(fontsize='small')
+plot_InorgN = numpy.zeros([nplots, nclays, nclimates])
+plot_SAPC = numpy.zeros([nplots, nclays, nclimates])
+plot_UnpC = numpy.zeros([nplots, nclays, nclimates])
+plot_ECMC = numpy.zeros([nplots, nclays, nclimates])
+plot_AMC = numpy.zeros([nplots, nclays, nclimates])
+plot_TotN = numpy.zeros([nplots, nclays, nclimates])
+plot_NfromNecro = numpy.zeros([nplots, nclays, nclimates])
+plot_NfromSOM = numpy.zeros([nplots, nclays, nclimates])
+plot_Nsource = numpy.zeros([nplots, nclays, nclimates])
+plot_Nlimit = numpy.zeros([nplots, nclays, nclimates])
+
+plot_InorgN_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_SAPC_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_UnpC_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_ECMC_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_AMC_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_TotN_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_NfromNecro_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_NfromSOM_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_Nsource_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_Nlimit_all = numpy.zeros([timesteps, nplots, nclays, nclimates])
+plot_Nlimit_acc = numpy.zeros([timesteps, nplots, nclays, nclimates])
+for plotnum in range(nplots):
+    for claynum in range(nclays):
+        for climnum in range(nclimates):
+            filename = "/Users/f0068s6/PycharmProjects/CORPSE_Siya/" + str(
+                nclimates*nclays*plotnum + nclimates*claynum + climnum + 1) + "_Quarterly_data.txt"
+            file = open(filename, "r")
+            datastring = file.read()
+            datalist = datastring.split("\n")
+            for i in range(0, timesteps, 1):
+                data = datalist[i].split(" ")
+                plot_InorgN_all[i, plotnum, claynum, climnum] = data[0]
+                plot_SAPC_all[i, plotnum, claynum, climnum] = data[1]
+                plot_UnpC_all[i, plotnum, claynum, climnum] = data[2]
+                plot_ECMC_all[i, plotnum, claynum, climnum] = data[3]
+                plot_AMC_all[i, plotnum, claynum, climnum] = data[4]
+                plot_TotN_all[i, plotnum, claynum, climnum] = data[5]
+                plot_NfromNecro_all[i, plotnum, claynum, climnum] = data[6]
+                plot_NfromSOM_all[i, plotnum, claynum, climnum] = data[7]
+                if i==0:
+                    plot_Nsource_all[i, plotnum, claynum, climnum] = float(data[6])/float(data[7])
+                else:
+                    plot_Nsource_all[i, plotnum, claynum, climnum] = \
+                        (float(data[6])-plot_NfromNecro_all[i-1, plotnum, claynum, climnum])/(float(data[7])-plot_NfromSOM_all[i-1, plotnum, claynum, climnum])
+                plot_Nlimit_acc[i, plotnum, claynum, climnum] = data[8]
```

Comment:
You could load this data with `pandas.DataFrame.to_csv(filename)`

Reply:
Sorry, I meant `pandas.read_csv`. Or you could use `numpy.loadtxt` to read it into a numpy array instead of a pandas DataFrame